src/open_inwoner/cms/cases/views/services.py

```diff
@@ -53,52 +61,49 @@ def identification(self) -> str:
     def process_data(self) -> dict:
         return {**self.submission.process_data(), "api_group": self.api_group}
 
+    def __hash__(self):
+        return hash((self.identification, self.api_group.pk))
+
 
-class ThreadLimits(TypedDict):
-    zgw_api_groups: int
-    resolve_case_list: int
-    resolve_case_instance: int
+class Timeouts(TypedDict):
+    fetch_raw_cases: int | float
+    resolve_cases: int | float
+    fetch_submissions: int | float
 
 
 class CaseListService:
     request: HttpRequest
-    _thread_limits: ThreadLimits
-    _thread_timeouts: ThreadLimits
+    _max_workers: int
+    _timeouts: Timeouts
 
     def __init__(self, request: HttpRequest):
         self.request = request
-        self._thread_timeouts = {
-            "zgw_api_groups": 60,
-            "resolve_case_instance": 15,
-            "resolve_case_list": 15,
+        self._timeouts = {
+            "fetch_raw_cases": settings.ZGW_CASE_LIST_FETCH_TIMEOUT * 0.3,
+            "resolve_cases": settings.ZGW_CASE_LIST_FETCH_TIMEOUT * 0.5,
+            "fetch_submissions": settings.ZGW_CASE_LIST_FETCH_TIMEOUT * 0.2,
```

Comment:
What's the rationale for multiplying the values with a constant instead of taking them as they are? No objection, just curious.